tests/errorParser.test.ts

```diff
@@ -63,10 +73,29 @@ describe('errorParser', () => {
 
   describe('Predefined Api errors', () => {
     it('Should succesfully parse default ApiError', () => {
+      const errorTranslation = 'English translation';
+      i18nMock.mockImplementation(() => {
```

Comment:
Can be one line due to arrow fn
`i18nMock.mockImplementation(() => errorTranslation)`